solidity/dashboard/package.json

```diff
@@ -45,16 +45,19 @@
     "build": "npm run build-css && react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "js:lint": "eslint src/**/*.js src/**/*.jsx",
-    "js:lint:fix": "eslint --fix src/**/*.js src/**/*.jsx"
+    "lint": "eslint --ext .jsx --ext .js .",
+    "lint:fix": "eslint --fix --ext .jsx --ext .js .",
+    "format": "npm run lint && prettier --check .",
+    "format:fix": "npm run lint:fix && prettier --write ."
   },
   "devDependencies": {
+    "@keep-network/prettier-config-keep": "github:keep-network/prettier-config-keep#a1a333e",
     "@redux-saga/testing-utils": "^1.1.3",
     "@testing-library/react-hooks": "^5.1.2",
     "@types/jest": "^26.0.21",
     "eslint": "^6.8.0",
-    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git",
-    "prettier": "^2.0.4",
+    "eslint-config-keep": "github:keep-network/eslint-config-keep#0c27ade",
+    "prettier": "^2.3.2",
```

Comment:
We'll also need the `"prettier-plugin-sh": "^0.6.1",` dependency if `prettier-plugin-sh` will be added to `solidity/dashboard/.prettierrc.js` according to Kuba's suggestion.